### ISSUE TITLE
Obtaining Plaid developer account

### ISSUE THREAD

**Issue by TRSx80** [ebridges/plaid2qif#12]
Good day! And thanks for sharing this tool! :)

Did anyone have issues with this?

I signed up a while back, created an account. Which gets you to "Sandbox" level, which if I am understanding correctly, does not allow you to actually do anything live with any of your accounts. For that you need the next level up from that (I forget what they call it) which is still free, but enables real functionality with banks.

I sent the request in, but never heard anything back. So recently I followed up with them (Plaid) and they are now asking for detailed instructions about what I plan to build.

Personally I am strongly against fibbing, but I am worried that if I tell them I plan on personal use only and explain what plain text accounting is, that they might not grant me the developer account.

So I was just curious how others handled this?

**Comment by ebridges** [comment]
Have you tried with a sandbox account?  That’s what I use and it works fine.  That said, I just use it for personal finance needs. I’m not sure what your goals are. If you’re using the plaid account for a business then you would need to provide them more information.

**Comment by TRSx80** [comment]
Oh no, just trying to get it working for my own stuff.

Thanks for the info! I'm glad I asked. I'll try and get it working with just the "Sandbox" level account.

**Comment by ebridges** [comment]
@TRSx80 were you able to get this to work with your bank?  Perhaps your issues are related to #16?

**Comment by TRSx80** [comment]
Thanks for the suggestion, ebridges.

In the meantime, I gave up on the Plaid approach.  If I'm being honest, I never really liked the idea of depending on a third party API in my work flow anyway (without even getting into security concerns).

At one point I got an OFX based approach working (after trying 2-3 times over period of 1-2 years maybe).  Then I realized that the data was very ugly, all caps, etc.

So in the end, now I just (somewhat)[1] manually enter my payees and narration, the former based on a "controlled vocabulary" list I maintain in a simple text file.  I think I finally just realized that I will never get a nice looking, properly capitalized, and (perhaps most importantly) consistent payee using automated tools.  Much less any sort of meaningful narration.

Data ultimately gets into Beancount, btw.

[1] What I actually ended up doing is developing a workflow that starts from taking photos of receipts.  Then I (quite recently in fact) whipped up some custom Emacs lisp which act as helper functions around `ivy-read`.  So I look at the receipts in `dired` and then invoke my custom function, which prompts me for all the info (payee, narration, amount, etc.) and then not only automatically renames the receipt photo filename at point, but also automatically generates the Beancount transaction in a separate buffer.  Thereby needing to only "touch" each receipt once, and get everything I need from it in a mostly automated way.

Not sure why I mention all of that here, maybe someone finds it helpful I guess.  I have struggled for literally years to finally find an approach that works for me.

Also, I suppose I will close the issue, as I don't think that a "Developer" account is actually required to accomplish basic functionality for personal usage (even though I ended up going in a completely different direction).  Thanks for the reminder, ebridges, and cheers!  :beers:
